requirements/source/requirements-base.in

```diff
@@ -1,6 +1,6 @@
 boto3<2
 botocore<2
-django==4.2.11
+django==5.0
```

Comment:
should this not be django>5.0 or 5.0. something?

Reply:
Going by the ticket, I assumed we wanted to increment the Django version 'bit-by-bit'. 

By saying we want the Django version to be less than 6, what would the version of Django actually be? 

My next incremental update would have been to Django version 5.0.3 (which is the latest version of it now).

Reply:
* `5.0` is better than 4.2.11, so ok for this PR, as a temporary choice
* `<5.1` would be better still, and is the format we want in this source file, so that we more easily accept patch updates

Reply:
Hey both, thanks for reviewing. I hadn't realised we were going straight to the latest django version. Was following the ticket description of doing it "stepwise: 5.0, 5.1, 5.2". 

I've updated this to now be `django<5.1`